### PR TITLE
community: Add SparkLLM to community

### PR DESCRIPTION
**Pull request by liugddx**
Thank you for contributing to LangChain!

Checklist:

- [x] PR title: Please title your PR "package: description", where "package" is whichever of langchain, community, core, experimental, etc. is being modified. Use "docs: ..." for purely docs changes, "templates: ..." for template changes, "infra: ..." for CI changes.
  - Example: "community: add foobar LLM"
- [ x] PR message: **Delete this entire template message** and replace it with the following bulleted list
    - **Description:** community: Add SparkLLM to community
    - **Issue:** the issue # it fixes, if applicable
    - **Dependencies:** No
    - **Twitter handle:** No
- [x] Pass lint and test: Run `make format`, `make lint` and `make test` from the root of the package(s) you've modified to check that you're passing lint and testing. See contribution guidelines for more information on how to write/run tests, lint, etc: https://python.langchain.com/docs/contributing/
- [x] Add tests and docs: If you're adding a new integration, please include
  1. a test for the integration, preferably unit tests that do not rely on network access,
  2. an example notebook showing its use. It lives in `docs/docs/integrations` directory.

### CODE REVIEW

**Comment by liugddx** [comment]
@baskaryan Please help me review the code, thanks.

**Review by baskaryan — APPROVED** [review]

**Comment by liugddx** [comment]
@baskaryan PTAL, thanks.

**Comment by liugddx** [comment]
@baskaryan Please help me merge this pr.

**Review by baskaryan — APPROVED** [review]

File: libs/community/langchain_community/llms/__init__.py [hunk 1]
```diff
@@ -950,4 +959,5 @@ def get_type_to_cls_dict() -> Dict[str, Callable[[], Type[BaseLLM]]]:
         "yandex_gpt": _import_yandex_gpt,
         "yuan2": _import_yuan2,
         "VolcEngineMaasLLM": _import_volcengine_maas,
+        "SparkLLM": _import_sparkllm(),
```
Comment (drazvan) [inline libs/community/langchain_community/llms/__init__.py:962]: The `_import_sparkllm()` function should not be called here. It's breaking everything with the message:

"Did not find spark_app_id, please add an environment variable `IFLYTEK_SPARK_APP_ID` which contains it, or pass `spark_app_id` as a named parameter. (type=value_error)"

Comment (liugddx) [reply libs/community/langchain_community/llms/__init__.py:962]: > The `_import_sparkllm()` function should not be called here. It's breaking everything with the message:
> 
> "Did not find spark_app_id, please add an environment variable `IFLYTEK_SPARK_APP_ID` which contains it, or pass `spark_app_id` as a named parameter. (type=value_error)"

Thanks @drazvan , I will fix it.

**Pull request merged**